bin.src/validateDrp.py

```diff
@@ -52,6 +52,8 @@
                                      formatter_class=argparse.RawDescriptionHelpFormatter)
     parser.add_argument('repo', type=str,
                         help='path to a repository containing the output of processCcd')
+    parser.add_argument('--outputPrefix', '-o', type=str, default=None,
+                        help='Define basic name prefix for output files.  Can include paths.  E.g., --outputPrefix="mydir/awesome_reduction" will produce "mydir/awesome_reduction_r.json" for the r-band JSON file.')
```

Comment:
This is longer than 110 char

Reply:
Also, don't we need to pass this to `validate.run()`?

Reply:
Oops!  Yes.  Done.